topics/epigenetics/tutorials/hicexplorer/tutorial.md

```diff
@@ -0,0 +1,263 @@
+
+TAD calling works in two steps: First HiCExplorer computes a TAD-separation score based on a z-score matrix for all bins. Then those bins having a local minimum of the TAD-separation score are evaluated with respect to the surrounding bins to decide assign a p-value. Then a cutoff is applied to select the bins more likely to be TAD boundaries.
+
+[hicFindTADs](http://hicexplorer.readthedocs.io/en/latest/content/tools/hicFindTADs.html#hicfindtads) tries to identify sensible parameters but those can be change to identify more stringent set of boundaries.
+
+> ### {% icon hands_on %} Hands-on: Finding TADs
+>
+> 1. **hicFindTADs** {% icon tool %}: Run hicFindTADs on the corrected matrix adjusting the parameters:
+>    - "Minimum window length (in bp) to be considered to the left and to the right of each Hi-C bin." to `30000`
+>    - "Maximum window length (in bp) to be considered to the left and to the right of each Hi-C bin." to `60000`
+>    - "Step size when moving from minDepth to maxDepth" to `10000`
+>    - "Multiple Testing Corrections" to `False discovery rate`
+>    - "q-value" to `0.05` 
+>    - "Minimum threshold of the difference between the TAD-separation score of a putative boundary and the mean of the TAD-sep. score of surrounding bins." to `0.05`
+>
+{: .hands_on}
+
+As an output we get the boundaries, domains and scores separated files. We will use in the plot later only the TAD-score file.
+
+We can plot the TADs for a given chromosomal region. For this we will use [hicPlotTADs](http://hicexplorer.readthedocs.io/en/latest/content/tools/hicPlotTADs.html). But before make sure to import [gene track file](https://zenodo.org/record/1176070/files/dm6_genes.bed) in .bed format from [Zenodo](https://doi.org/10.5281/zenodo.1176070).
+
+
+> ### {% icon hands_on %} Hands-on: Plotting TADs
+>
+> 1. **hicPlotTADs** {% icon tool %}: Run hicPlotTADs adjusting the parameters:
+>    - "Region of the genome to limit the operation" to `chr2L:1000000-5000000`
+>    - "Choose style of the track" to `TAD visualization`
+>    - "Plot title" to `HiC dm6 chr2L:1000000-5000000`
+>    - "Matrix to compute on." to the corrected matrix from hicCorrectMatrix step
+>    - "Depth" to `2000000`
+>    - "Plotting type" to `interaction`
+>    - "Boundaries file" to the output from hicFindTADs
+>    - "Show x labels" to `Yes`
+>    - "+Insert Include tracks in your plot"
+>    - "Choose style of the track" to `TAD score`
+>    - "Plot title" to `TAD separation score`
+>    - "Track file bedgraph format" to the output from previous step
+>    - "Color of track" to blue
+>    - "Width" to `2`
+>    - "+Insert Include tracks in your plot"
+>    - "Choose style of the track" to `Gene track`
+>    - "Plot title" to `dm6 genes`
+>    - "Track file bedgraph format" the imported .bed file

```

Comment:
again many possbilities

Reply:
Torsten, could you open a new PR with these changes?

Reply:
... :neutral_face: